article/app/views/fragments/emailArticleBody.scala.html

```diff
@@ -0,0 +1,20 @@
+
+@(page: ArticlePage)(implicit request: RequestHeader)
+
+@import common.LinkTo
+@import views.BodyCleaner
+@import views.support.TrailCssClasses.toneClass
+@import conf.switches.Switches._
+
+@defining(page.article) { article =>
+    @fragments.headTonal(article, article.commercial.isAdvertisementFeature && NewCommercialContent.isSwitchedOn, showMeta = false)
+
+    @fragments.emailMainMedia(article)
+
+    @fragments.contentMeta(article, showExtras = false)
+
+    <div class="content__article-body from-content-api" itemprop="@if(article.tags.isReview){reviewBody} else {articleBody}"
```

Comment:
Very minor & for future updates: I am not sure whether itemProp would be useful within the email template but maybe some like the [Schema](https://developers.google.com/schemas/reference/) would work nicely in the email (orani**z**ation for example).